atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/persistentlock/LockStoreTest.java

```diff
@@ -58,6 +61,14 @@ public void lockIsInitiallyOpen() {
     }
 
     @Test
+    public void noErrorIfLockOpenedWhileCreatingTable() {
+        doThrow(new CheckAndSetException("foo", null, null, ImmutableList.of())).when(kvs).checkAndSet(anyObject());
+
+        System.out.println("test");
```

Comment:
Took me a moment to figure out what you meant by this (at first I thought "see previous comment", but could have been "wanted to type ^^ but failed").

Deleting the "test" line.